diff --git a/sqlite3/sqlite3.c b/sqlite3/sqlite3.c
index aa96357..310583f 100644
--- a/sqlite3/sqlite3.c
+++ b/sqlite3/sqlite3.c
@@ -41240,16 +41240,16 @@ SQLITE_API int sqlite3_fullsync_count = 0;
 #if !defined(fdatasync) && !HAVE_FDATASYNC
 # define fdatasync fsync
 #endif
 
 /*
 ** Define HAVE_FULLFSYNC to 0 or 1 depending on whether or not
-** the F_BARRIERFSYNC macro is defined.  F_BARRIERFSYNC is currently
+** the F_FULLFSYNC macro is defined.  F_FULLFSYNC is currently
 ** only available on Mac OS X.  But that could change.
 */
-#ifdef F_BARRIERFSYNC       // JHB: Search and replaced F_FULLFSYNC with F_BARRIERFSYNC
+#ifdef F_FULLFSYNC
 # define HAVE_FULLFSYNC 1
 #else
 # define HAVE_FULLFSYNC 0
 #endif
 
 
@@ -41313,13 +41313,13 @@ static int full_fsync(int fd, int fullSync, int dataOnly){
   {
     struct stat buf;
     rc = osFstat(fd, &buf);
   }
 #elif HAVE_FULLFSYNC
   if( fullSync ){
-    rc = osFcntl(fd, F_BARRIERFSYNC, 0);
+    rc = osFcntl(fd, F_FULLFSYNC, 0);
   }else{
     rc = 1;
   }
   /* If the FULLFSYNC failed, fall back to attempting an fsync().
   ** It shouldn't be possible for fullfsync to fail on the local
   ** file system (on OSX), so failure indicates that FULLFSYNC
@@ -59783,13 +59783,13 @@ SQLITE_PRIVATE void sqlite3PagerShrink(Pager *pPager){
 ** file is synced following each commit operation, in addition to the
 ** syncs associated with NORMAL.  There is no difference between FULL
 ** and EXTRA for WAL mode.
 **
 ** Do not confuse synchronous=FULL with SQLITE_SYNC_FULL.  The
 ** SQLITE_SYNC_FULL macro means to use the MacOSX-style full-fsync
-** using fcntl(F_BARRIERFSYNC).  SQLITE_SYNC_NORMAL means to do an
+** using fcntl(F_FULLFSYNC).  SQLITE_SYNC_NORMAL means to do an
 ** ordinary fsync() call.  There is no difference between SQLITE_SYNC_FULL
 ** and SQLITE_SYNC_NORMAL on platforms other than MacOSX.  But the
 ** synchronous=FULL versus synchronous=NORMAL setting determines when
 ** the xSync primitive is called and is relevant to all platforms.
 **
 ** Numeric values associated with these states are OFF==1, NORMAL=2,
